index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Nemwel Boniface">
+    <meta name="description" content="This is my personal portfolio site that I will share to all potential employers to showcase my skills as a developer">
+
+    <!--google Fonts roboto, crete-round-->
+    
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+
+    <!-- font awesome icons -->
+
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Nemwel Portfolio</title>
+
+    <!--Page stylesheets-->
+    <link rel="stylesheet" href="stylesheets/landing.css">
+
+    <!--Smooth scroll script-->
+       
+    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll/dist/smooth-scroll.polyfills.min.js"></script>
+
+</head>
+<body>
+    <header class="header">
+        <div class="topHeaderBar">
```

Comment:
- [ ] This element has a top margin is creating a white gap on the top of the page.  If you correct this margin, the page will fit perfectly on the top of the device's screen.  💪

Reply:
Hello @Meltrust  Thank you for this review.
I have changed the top margin for my class="topHeaderBar" as you had suggested.
I however did not understand the second suggestion. The hamburger button is inside a nav tag . Kindly clarify to me this bit. I have attached a screenshot of how the code is structured.
Thank you
![NAVBAR](https://user-images.githubusercontent.com/86318284/148387024-e668c8c0-e15e-4c0f-b776-778a1aa924a1.png)
u

Reply:
Hi Nemwel,

I investigated and double-checked with my superiors for clarification on this particular case.

The answer is:

This particular `<div>` kindly please change it to `<nav>`: 

![image](https://user-images.githubusercontent.com/57421823/148394877-6a7c0cf9-3087-4ce4-baf6-fee8bd69de8c.png)


I hope this is clear now 👍

Reply:
@Meltrust  Okay thank you I will change it right away!